### DISCUSSION TITLE
Temperature Feeds from REM to Nixie

### DISCUSSION THREAD

**Discussion by dougcfl** [tagyoureit/nodejs-poolController#829] [Nixie] [answered]
Hello.  I have 4 temperature probes available to feed from REM to Nixie; Pool, SPA, ambient air, and internal-to-enclosure.  I have a gas heater, but no solar in a shared body system.  I have the sensors fed through an ADC then mapped to a generic device and all 4 are working.  Where do the various sensor feeds from REM (the feed from the Generic Sensor definitions to njsPC) display in the Nixie dashpanel? I have one Generic Device mapped to waterSensor1 and that looks like the Pool Temp.  I have the ambient air temp mapped to airSensor and that is Air Temp.  The SPA temp in Nixie is not the same as the Pool temp.  Where's that coming from?
Can someone complete the map from REM Generic Device Feed to Nixie?
temps.waterSensor1 displays as Pool temp
temps.waterSensor2 displays as ??
temps.waterSensor3 displays as ??
temps.waterSensor4 displays as ??
temps.airSensor displays as Air Temp
temps.solarSensor1 displays as ??
temps.solarSensor2 displays as ??
temps.solarSensor3 displays as ??
temps.solarSensor4displays as ??
Thank you!

**Comment by rstrouse** [comment, marked as answer]
For a shared body system you should have 1 water sensor and 1 air temp sensor.  The 2,3,4 sensors are for multi-body systems.  If you do not have solar then you do not need a solar temp sensor.  This would be placed on the roof next to the panels to ensure there is enough heat in them to operate.  I'm not sure what 4 temperature sensors will do for you.   

In a shared body system the waterSensor1 probe should be placed just after the pump and before the filter.  You do not want it before the pump since the seal is for pressure not vacuum or suction.  You also do not want it after the heater as this will just drive you insane as the heater will not be able to heat to the source temperature.  This will handle all of the heater control from this one sensor whether you are heating the pool or the spa.
